--- src/javafx.base/module-info.java	2026-01-27 06:40:44
+++ src/javafx.base/module-info.java	2026-02-24 10:13:58
@@ -31,7 +31,7 @@
  * @since 9
  */
 module javafx.base {
-    requires java.desktop;
+    requires jfx.desktop;
     requires static jdk.jfr;
 
     exports javafx.beans;
@@ -54,37 +54,24 @@
         // END: incubator placeholder
         jfx.incubator.input,
         jfx.incubator.richtext,
-        javafx.graphics,
-        javafx.fxml,
-        javafx.media,
-        javafx.swing,
-        javafx.web;
+        javafx.graphics;
     exports com.sun.javafx.beans to
         javafx.controls,
-        javafx.fxml,
         javafx.graphics;
     exports com.sun.javafx.binding to
         javafx.controls,
         javafx.graphics;
     exports com.sun.javafx.collections to
         javafx.controls,
-        javafx.graphics,
-        javafx.media,
-        javafx.swing;
+        javafx.graphics;
     exports com.sun.javafx.event to
         javafx.controls,
         javafx.graphics;
     exports com.sun.javafx.logging to
         javafx.controls,
-        javafx.graphics,
-        javafx.fxml,
-        javafx.swing,
-        javafx.web;
+        javafx.graphics;
     exports com.sun.javafx.property to
         javafx.controls;
-    exports com.sun.javafx.reflect to
-        javafx.fxml,
-        javafx.web;
     exports com.sun.javafx.runtime to
         javafx.graphics;
 }
