--- src/javafx.graphics/com/sun/javafx/iio/jpeg/JPEGImageLoader.java	2026-01-27 06:40:48
+++ src/javafx.graphics/com/sun/javafx/iio/jpeg/JPEGImageLoader.java	2026-02-24 11:57:40
@@ -28,7 +28,6 @@
 import com.sun.javafx.iio.ImageFrame;
 import com.sun.javafx.iio.ImageMetadata;
 import com.sun.javafx.iio.ImageStorage.ImageType;
-import com.sun.glass.utils.NativeLibLoader;
 import com.sun.javafx.iio.common.ImageLoaderImpl;
 import com.sun.javafx.iio.common.ImageTools;
 import java.io.IOException;
@@ -101,7 +100,7 @@
     private native boolean decompressIndirect(long structPointer, boolean reportProgress, byte[] array) throws IOException;
 
     static {
-        NativeLibLoader.loadLibrary("javafx_iio");
+        // TODO: NativeLibLoader.loadLibrary("javafx_iio");
 
         initJPEGMethodIDs(InputStream.class);
     }
