--- src/javafx.graphics/com/sun/javafx/util/Utils.java	2026-01-27 06:40:48
+++ src/javafx.graphics/com/sun/javafx/util/Utils.java	2026-02-24 17:35:20
@@ -45,8 +45,6 @@
 import javafx.stage.Window;
 import java.math.BigDecimal;
 import java.util.List;
-import com.sun.javafx.PlatformUtil;
-import com.sun.glass.utils.NativeLibLoader;
 import com.sun.prism.impl.PrismSettings;
 
 /**
@@ -921,21 +919,21 @@
      * Returns true if the operating system is a form of Windows.
      */
     public static boolean isWindows(){
-        return PlatformUtil.isWindows();
+        return false; // TODO: PlatformUtil.isWindows();
     }
 
     /**
      * Returns true if the operating system is a form of Mac OS.
      */
     public static boolean isMac(){
-        return PlatformUtil.isMac();
+        return false; // TODO: PlatformUtil.isMac();
     }
 
     /**
      * Returns true if the operating system is a form of Unix, including Linux.
      */
     public static boolean isUnix(){
-        return PlatformUtil.isUnix();
+        return false; // TODO: PlatformUtil.isUnix();
     }
 
     /***************************************************************************
@@ -1021,7 +1019,7 @@
         if (PrismSettings.verbose) {
             System.out.println("Loading Prism common native library ...");
         }
-        NativeLibLoader.loadLibrary(libName);
+        // TODO: NativeLibLoader.loadLibrary(libName);
         if (PrismSettings.verbose) {
             System.out.println("\tsucceeded.");
         }
