--- src/javafx.controls/com/sun/javafx/scene/control/skin/Utils.java	2026-01-27 06:40:46
+++ src/javafx.controls/com/sun/javafx/scene/control/skin/Utils.java	2026-02-25 09:08:34
@@ -841,7 +841,7 @@
 
     public static void addMnemonics(ContextMenu popup, Scene scene, boolean initialState, List<Mnemonic> into) {
 
-        if (!com.sun.javafx.PlatformUtil.isMac()) {
+        if (!false) { // TODO: com.sun.javafx.PlatformUtil.isMac()
 
             ContextMenuContent cmContent = (ContextMenuContent)popup.getSkin().getNode();
             MenuItem menuitem;
@@ -873,7 +873,7 @@
 
     public static void removeMnemonics(ContextMenu popup, Scene scene) {
 
-        if (!com.sun.javafx.PlatformUtil.isMac()) {
+        if (!false) { // TODO: com.sun.javafx.PlatformUtil.isMac()
 
             ContextMenuContent cmContent = (ContextMenuContent)popup.getSkin().getNode();
             MenuItem menuitem;
