--- src/javafx.graphics/com/sun/javafx/tk/Toolkit.java	2026-01-27 06:40:48
+++ src/javafx.graphics/com/sun/javafx/tk/Toolkit.java	2026-02-24 17:34:04
@@ -66,8 +66,6 @@
 import java.util.concurrent.Future;
 import com.sun.glass.ui.CommonDialogs.FileChooserResult;
 import com.sun.glass.ui.GlassRobot;
-import com.sun.glass.utils.NativeLibLoader;
-import com.sun.javafx.PlatformUtil;
 import com.sun.javafx.beans.event.AbstractNotifyListener;
 import com.sun.javafx.embed.HostInterface;
 import com.sun.javafx.geom.Path2D;
@@ -166,7 +164,7 @@
     public static synchronized void loadMSWindowsLibraries() {
         for (String libName : msLibNames) {
             try {
-                NativeLibLoader.loadLibrary(libName);
+                // TODO: NativeLibLoader.loadLibrary(libName);
             } catch (Throwable t) {
                 if (verbose) {
                     System.err.println("Error: failed to load "
@@ -177,17 +175,7 @@
     }
 
     private static String getDefaultToolkit() {
-        if (PlatformUtil.isWindows()) {
-            return DEFAULT_TOOLKIT;
-        } else if (PlatformUtil.isMac()) {
-            return DEFAULT_TOOLKIT;
-        } else if (PlatformUtil.isLinux()) {
-            return DEFAULT_TOOLKIT;
-        } else if (PlatformUtil.isIOS()) {
-            return DEFAULT_TOOLKIT;
-        } else if (PlatformUtil.isAndroid()) {
-           return DEFAULT_TOOLKIT;
-        }
+        // TODO: PlatformUtil.is()...
 
         throw new UnsupportedOperationException(System.getProperty("os.name") + " is not supported");
     }
@@ -202,7 +190,7 @@
         VersionInfo.setupSystemProperties();
 
         // Load required Microsoft runtime DLLs on Windows platforms
-        if (PlatformUtil.isWindows()) {
+        if (false) { // TODO: PlatformUtil.isWindows()
             loadMSWindowsLibraries();
         }
 
@@ -813,7 +801,7 @@
      * @return the key code for shortcut modifier key
      */
     public KeyCode getPlatformShortcutKey() {
-        return PlatformUtil.isMac() ? KeyCode.META : KeyCode.CONTROL;
+        return null; // TODO: PlatformUtil.isMac() ? KeyCode.META : KeyCode.CONTROL;
     }
 
     /**
