--- src/javafx.graphics/com/sun/javafx/font/DFontDecoder.java	2026-01-27 06:40:48
+++ src/javafx.graphics/com/sun/javafx/font/DFontDecoder.java	2026-02-24 11:39:28
@@ -27,11 +27,9 @@
 
 import java.io.IOException;
 
-import com.sun.glass.utils.NativeLibLoader;
-
 class DFontDecoder extends FontFileWriter {
     static {
-        NativeLibLoader.loadLibrary("javafx_font");
+        // TODO: NativeLibLoader.loadLibrary("javafx_font");
     }
     private native static long createCTFont(String fontName);
     private native static void releaseCTFont(long font);
