--- sdk/javafx.graphics/com/sun/javafx/font/LogicalFont.java	2021-10-16 18:21:20.000000000 -0400
+++ src/javafx.graphics/com/sun/javafx/font/LogicalFont.java	2021-12-27 17:32:50.182095637 -0500
@@ -180,8 +180,6 @@
                 FontConfigManager.getFontConfigFont(family, bold, italic);
             physicalFullName = fcCompFont.firstFont.fullName;
             physicalFileName = fcCompFont.firstFont.fontFile;
-        } else {
-            physicalFamily = PrismFontFactory.getSystemFont(familyName);
         }
     }
 
