--- src/javafx.graphics/com/sun/javafx/font/PrismFontFactory.java	2026-01-27 06:40:48
+++ src/javafx.graphics/com/sun/javafx/font/PrismFontFactory.java	2026-02-26 16:15:03
@@ -38,8 +38,6 @@
 import java.util.Locale;
 
 import com.sun.glass.ui.Screen;
-import com.sun.glass.utils.NativeLibLoader;
-import com.sun.javafx.PlatformUtil;
 import com.sun.javafx.text.GlyphLayout;
 
 public abstract class PrismFontFactory implements FontFactory {
@@ -79,15 +77,15 @@
     HashMap<String, CompositeFontResource> compResourceMap = new HashMap<>();
 
     static {
-        isWindows = PlatformUtil.isWindows();
-        isMacOSX  = PlatformUtil.isMac();
-        isLinux   = PlatformUtil.isLinux();
-        isIOS     = PlatformUtil.isIOS();
-        isAndroid = PlatformUtil.isAndroid();
-        isEmbedded = PlatformUtil.isEmbedded();
+        isWindows = false; // TODO: PlatformUtil.isWindows();
+        isMacOSX  = false; // TODO: PlatformUtil.isMac();
+        isLinux   = false; // TODO: PlatformUtil.isLinux();
+        isIOS     = false; // TODO: PlatformUtil.isIOS();
+        isAndroid = false; // TODO: PlatformUtil.isAndroid();
+        isEmbedded = false; // TODO: PlatformUtil.isEmbedded();
         int[] tempCacheLayoutSize = {0x10000};
 
-        NativeLibLoader.loadLibrary("javafx_font");
+        // TODO: NativeLibLoader.loadLibrary("javafx_font");
         String dbg = System.getProperty("prism.debugfonts", "");
         debugFonts = "true".equals(dbg);
         jreFontDir = getJDKFontDir();
@@ -488,7 +486,7 @@
          */
         if (isWindows) {
             int style = ((bold ? 1 : 0)) + ((italic) ? 2 : 0);
-            String fontFile = WindowsFontMap.findFontFile(lcFamilyName, style);
+            String fontFile = "?"; // TODO: WindowsFontMap.findFontFile(lcFamilyName, style);
             if (fontFile != null) {
                 fr = createFontResource(null, fontFile);
                 if (fr != null) {
@@ -640,35 +638,7 @@
         /* Still need decode the dfont (event when coretext is used)
          * so that JFXFontFont can read it */
         if (isMacOSX) {
-            DFontDecoder decoder = null;
-            if (name != null) {
-                if (file.endsWith(".dfont")) {
-                    decoder = new DFontDecoder();
-                    try {
-                        decoder.openFile();
-                        decoder.decode(name);
-                        decoder.closeFile();
-                        file = decoder.getFile().getPath();
-                    } catch (Exception e) {
-                        file = null;
-                        decoder.deleteFile();
-                        decoder = null;
-                        if (PrismFontFactory.debugFonts) {
-                            e.printStackTrace();
-                        }
-                    }
-                }
-            }
-            if (file != null) {
-                fr = createFontResource(name, file);
-            }
-            if (decoder != null) {
-                if (fr != null) {
-                    addDecodedFont(fr);
-                } else {
-                    decoder.deleteFile();
-                }
-            }
+            // TODO: DFontDecoder support
         } else {
             fr = createFontResource(name, file);
         }
@@ -726,7 +696,7 @@
          */
         if (isWindows && name != null) {
             String lcName = name.toLowerCase();
-            String fontFile = WindowsFontMap.findFontFile(lcName, -1);
+            String fontFile = "?"; // TODO: WindowsFontMap.findFontFile(lcName, -1);
             if (fontFile != null) {
                 fr = createFontResource(null, fontFile);
                 if (fr != null) {
@@ -1652,10 +1622,7 @@
                 }
 
             } else if (isMacOSX || isIOS) {
-                MacFontFinder.populateFontFileNameMap(tmpFontToFileMap,
-                                                      fontToFamilyNameMap,
-                                                      familyToFontListMap,
-                                                      Locale.ENGLISH);
+                // TODO: MacFontFinder.populateFontFileNameMap(...
 
                 if (debugFonts) {
                     logFontInfo(" *** MACOS LOCATED FONTS:",
@@ -1675,10 +1642,7 @@
                                 familyToFontListMap);
                 }
             } else if (isAndroid) {
-               AndroidFontFinder.populateFontFileNameMap(tmpFontToFileMap,
-                        fontToFamilyNameMap,
-                        familyToFontListMap,
-                        Locale.ENGLISH);
+                // TODO: AndroidFontFinder.populateFontFileNameMap(...
            } else { /* unrecognised OS */
                 fontToFileMap = tmpFontToFileMap;
                 return fontToFileMap;
@@ -1695,7 +1659,7 @@
             fontToFileMap = tmpFontToFileMap;
             if (isAndroid) {
                 populateFontFileNameMapGeneric(
-                       AndroidFontFinder.getSystemFontsDir());
+                       null ); // TODO: AndroidFontFinder.getSystemFontsDir()
             }
             populateFontFileNameMapGeneric(jreFontDir);
 
@@ -1818,9 +1782,9 @@
             if (isWindows) {
                 systemFontSize = getSystemFontSizeNative();
             } else if (isMacOSX || isIOS) {
-                systemFontSize = MacFontFinder.getSystemFontSize();
+                systemFontSize = -1; // TODO: MacFontFinder.getSystemFontSize();
             } else if (isAndroid) {
-               systemFontSize = AndroidFontFinder.getSystemFontSize();
+               systemFontSize = -1; // TODO: AndroidFontFinder.getSystemFontSize();
             } else if (isEmbedded) {
                 try {
                     int screenDPI = Screen.getMainScreen().getResolutionY();
@@ -1846,12 +1810,12 @@
                         systemFontFamily = "Arial"; // play it safe.
                     }
                 } else if (isMacOSX || isIOS) {
-                    systemFontFamily = MacFontFinder.getSystemFont();
+                    systemFontFamily = "?"; // TODO: MacFontFinder.getSystemFont();
                     if (systemFontFamily == null) {
                         systemFontFamily = "Lucida Grande";
                     }
                 } else if (isAndroid) {
-                   systemFontFamily = AndroidFontFinder.getSystemFont();
+                   systemFontFamily = "?"; // TODO: AndroidFontFinder.getSystemFont();
                 } else {
                     systemFontFamily = "Lucida Sans"; // for now.
                 }
