--- src/javafx.controls/javafx/scene/control/skin/LabeledSkinBase.java	2026-01-27 06:40:46
+++ src/javafx.controls/javafx/scene/control/skin/LabeledSkinBase.java	2026-02-25 09:10:52
@@ -967,7 +967,7 @@
 
             if (cleanText != null && cleanText.length() > 0
                     && mnemonicInfo != null
-                    && !com.sun.javafx.PlatformUtil.isMac()
+                    // TODO: && !com.sun.javafx.PlatformUtil.isMac()
                     && getSkinnable().isMnemonicParsing()) {
                 /*
                 ** the Labeled has a MnemonicParsing property,
