--- src/javafx.controls/javafx/scene/control/skin/TextInputControlSkin.java	2026-01-27 06:40:46
+++ src/javafx.controls/javafx/scene/control/skin/TextInputControlSkin.java	2026-02-25 09:15:18
@@ -30,16 +30,12 @@
 import java.util.Collections;
 import java.util.List;
 
-import com.sun.javafx.PlatformUtil;
 import com.sun.javafx.scene.control.Properties;
 import com.sun.javafx.scene.control.behavior.TextInputControlBehavior;
-import com.sun.javafx.scene.control.skin.FXVK;
 import com.sun.javafx.scene.input.ExtendedInputMethodRequests;
 import com.sun.javafx.tk.FontMetrics;
 import com.sun.javafx.tk.Toolkit;
 
-import static com.sun.javafx.PlatformUtil.*;
-
 import javafx.animation.Animation.Status;
 import javafx.animation.KeyFrame;
 import javafx.animation.Timeline;
@@ -232,7 +228,7 @@
             @Override protected boolean computeValue() {
                 // JDK-8111037: On Windows, we show the caret during selection, but on others we hide it
                 return !blinkProperty().get() && displayCaret.get() && control.isFocused() &&
-                        (isWindows() || (control.getCaretPosition() == control.getAnchor())) &&
+                        (false || (control.getCaretPosition() == control.getAnchor())) && // TODO: isWindows()
                         !control.isDisabled() &&
                         control.isEditable();
             }
@@ -247,7 +243,7 @@
             selectionHandle1.setManaged(false);
             selectionHandle2.setManaged(false);
 
-            if (PlatformUtil.isIOS()) {
+            if (false) { // TODO: PlatformUtil.isIOS()) {
                 caretHandle.setVisible(false);
             } else {
                 caretHandle.visibleProperty().bind(new BooleanBinding() {
@@ -298,31 +294,7 @@
             selectionHandle2.setId("selection-handle-2");
         }
 
-        if (IS_FXVK_SUPPORTED) {
-            if (preload) {
-                Scene scene = control.getScene();
-                if (scene != null) {
-                    Window window = scene.getWindow();
-                    if (window != null) {
-                        FXVK.init(control);
-                    }
-                }
-            }
-            registerInvalidationListener(control.focusedProperty(), observable -> {
-                if (FXVK.useFXVK()) {
-                    Scene scene = getSkinnable().getScene();
-                    if (control.isEditable() && control.isFocused()) {
-                        FXVK.attach(control);
-                    } else if (scene == null ||
-                            scene.getWindow() == null ||
-                            !scene.getWindow().isFocused() ||
-                            !(scene.getFocusOwner() instanceof TextInputControl &&
-                                    ((TextInputControl)scene.getFocusOwner()).isEditable())) {
-                        FXVK.detach();
-                    }
-                }
-            });
-        }
+        // TODO: IS_FXVK_SUPPORTED
     }
 
     @Override
