--- src/javafx.graphics/com/sun/scenario/effect/impl/RendererFactory.java	2026-01-27 06:40:48
+++ src/javafx.graphics/com/sun/scenario/effect/impl/RendererFactory.java	2026-02-24 12:13:44
@@ -26,7 +26,6 @@
 package com.sun.scenario.effect.impl;
 
 import java.lang.reflect.Method;
-import com.sun.javafx.PlatformUtil;
 import com.sun.scenario.effect.FilterContext;
 
 /**
@@ -42,7 +41,7 @@
     private static boolean tryRSL = true;
     private static boolean trySIMD = false;
     // by default we only enable jogl hw acceleration on MacOS
-    private static boolean tryJOGL = PlatformUtil.isMac();
+    private static boolean tryJOGL = false; // TODO: PlatformUtil.isMac();
     private static boolean tryPrism = true;
 
     static {
