--- src/javafx.base/com/sun/javafx/reflect/ReflectUtil.java	2026-01-27 06:40:44
+++ src/javafx.base/com/sun/javafx/reflect/ReflectUtil.java	2026-02-24 10:13:58
@@ -26,15 +26,9 @@
 
 package com.sun.javafx.reflect;
 
-import com.sun.javafx.SecurityUtil;
 import java.lang.reflect.Proxy;
 
 public final class ReflectUtil {
-
-    static {
-        // Check for security manager (throws exception if enabled)
-        SecurityUtil.checkSecurityManager();
-    }
 
     private ReflectUtil() {
     }
