--- sdk/javafx.graphics/module-info.java	2021-10-16 18:21:20.000000000 -0400
+++ src/javafx.graphics/module-info.java	2021-12-27 17:32:50.220095204 -0500
@@ -40,8 +40,6 @@
  * @since 9
  */
 module javafx.graphics {
-    requires java.desktop;
-    requires java.xml;
     requires jdk.unsupported;
 
     requires transitive javafx.base;
@@ -75,25 +73,18 @@
     exports com.sun.javafx.application to
         java.base,
         javafx.controls,
-        javafx.swing,
         javafx.web;
     exports com.sun.javafx.css to
         javafx.controls;
-    exports com.sun.javafx.cursor to
-        javafx.swing;
-    exports com.sun.javafx.embed to
-        javafx.swing;
     exports com.sun.javafx.font to
         javafx.web;
     exports com.sun.javafx.geom to
         javafx.controls,
         javafx.media,
-        javafx.swing,
         javafx.web;
     exports com.sun.javafx.geom.transform to
         javafx.controls,
         javafx.media,
-        javafx.swing,
         javafx.web;
     exports com.sun.javafx.iio to
         javafx.web;
@@ -102,11 +93,9 @@
     exports com.sun.javafx.scene to
         javafx.controls,
         javafx.media,
-        javafx.swing,
         javafx.web;
     exports com.sun.javafx.scene.input to
         javafx.controls,
-        javafx.swing,
         javafx.web;
     exports com.sun.javafx.scene.layout to
         javafx.controls,
@@ -119,7 +108,6 @@
         javafx.web;
     exports com.sun.javafx.sg.prism to
         javafx.media,
-        javafx.swing,
         javafx.web;
     exports com.sun.javafx.stage to
         javafx.controls,
@@ -129,13 +117,10 @@
     exports com.sun.javafx.tk to
         javafx.controls,
         javafx.media,
-        javafx.swing,
         javafx.web;
     exports com.sun.javafx.util to
         javafx.controls,
-        javafx.fxml,
         javafx.media,
-        javafx.swing,
         javafx.web;
     exports com.sun.prism to
         javafx.media,
