--- src/javafx.graphics/com/sun/javafx/iio/ImageStorage.java	2026-01-27 06:40:48
+++ src/javafx.graphics/com/sun/javafx/iio/ImageStorage.java	2026-02-24 11:57:13
@@ -25,12 +25,10 @@
 
 package com.sun.javafx.iio;
 
-import com.sun.javafx.PlatformUtil;
 import com.sun.javafx.iio.ImageFormatDescription.Signature;
 import com.sun.javafx.iio.bmp.BMPImageLoaderFactory;
 import com.sun.javafx.iio.common.ImageTools;
 import com.sun.javafx.iio.gif.GIFImageLoaderFactory;
-import com.sun.javafx.iio.ios.IosImageLoaderFactory;
 import com.sun.javafx.iio.jpeg.JPEGImageLoaderFactory;
 import com.sun.javafx.iio.png.PNGImageLoaderFactory;
 import com.sun.javafx.logging.PlatformLogger;
@@ -182,7 +180,7 @@
     private Optional<ImageLoaderFactory> j2dImageLoaderFactory;
     private int maxSignatureLength;
 
-    private static final boolean isIOS = PlatformUtil.isIOS();
+    private static final boolean isIOS = false; // TODO: PlatformUtil.isIOS();
 
     private static class InstanceHolder {
         static final ImageStorage INSTANCE = new ImageStorage();
@@ -197,7 +195,7 @@
             //On iOS we have single factory/ native loader
             //for all image formats
             loaderFactories = new ImageLoaderFactory[]{
-                IosImageLoaderFactory.getInstance()
+                // TODO: IosImageLoaderFactory.getInstance()
             };
         } else {
             loaderFactories = new ImageLoaderFactory[]{
@@ -560,7 +558,7 @@
 
     private ImageLoader findImageLoader(InputStream stream, ImageLoadListener listener) throws IOException {
         if (isIOS) {
-            return IosImageLoaderFactory.getInstance().createImageLoader(stream);
+            return null; // TODO: IosImageLoaderFactory.getInstance().createImageLoader(stream);
         }
 
         // We need a stream that supports the mark and reset methods, since J2DImageLoader
