--- src/javafx.graphics/com/sun/javafx/iio/ImageStorage.java	2026-01-27 06:40:48
+++ src/javafx.graphics/com/sun/javafx/iio/ImageStorage.java	2026-02-26 15:51:47
@@ -25,13 +25,9 @@
 
 package com.sun.javafx.iio;
 
-import com.sun.javafx.PlatformUtil;
 import com.sun.javafx.iio.ImageFormatDescription.Signature;
-import com.sun.javafx.iio.bmp.BMPImageLoaderFactory;
 import com.sun.javafx.iio.common.ImageTools;
 import com.sun.javafx.iio.gif.GIFImageLoaderFactory;
-import com.sun.javafx.iio.ios.IosImageLoaderFactory;
-import com.sun.javafx.iio.jpeg.JPEGImageLoaderFactory;
 import com.sun.javafx.iio.png.PNGImageLoaderFactory;
 import com.sun.javafx.logging.PlatformLogger;
 import com.sun.javafx.util.DataURI;
@@ -182,7 +178,7 @@
     private Optional<ImageLoaderFactory> j2dImageLoaderFactory;
     private int maxSignatureLength;
 
-    private static final boolean isIOS = PlatformUtil.isIOS();
+    private static final boolean isIOS = false; // TODO: PlatformUtil.isIOS();
 
     private static class InstanceHolder {
         static final ImageStorage INSTANCE = new ImageStorage();
@@ -197,14 +193,12 @@
             //On iOS we have single factory/ native loader
             //for all image formats
             loaderFactories = new ImageLoaderFactory[]{
-                IosImageLoaderFactory.getInstance()
+                // TODO: IosImageLoaderFactory.getInstance()
             };
         } else {
             loaderFactories = new ImageLoaderFactory[]{
                 GIFImageLoaderFactory.getInstance(),
-                JPEGImageLoaderFactory.getInstance(),
                 PNGImageLoaderFactory.getInstance(),
-                BMPImageLoaderFactory.getInstance()
                 // Note: append ImageLoadFactory for any new format here.
             };
         }
@@ -560,7 +554,7 @@
 
     private ImageLoader findImageLoader(InputStream stream, ImageLoadListener listener) throws IOException {
         if (isIOS) {
-            return IosImageLoaderFactory.getInstance().createImageLoader(stream);
+            return null; // TODO: IosImageLoaderFactory.getInstance().createImageLoader(stream);
         }
 
         // We need a stream that supports the mark and reset methods, since J2DImageLoader
Only in tmp/javafx.graphics/com/sun/javafx/iio: bmp
Only in tmp/javafx.graphics/com/sun/javafx/iio: jpeg
