--- src/javafx.controls/com/sun/javafx/scene/control/behavior/TableViewBehaviorBase.java	2026-01-27 06:40:46
+++ src/javafx.controls/com/sun/javafx/scene/control/behavior/TableViewBehaviorBase.java	2026-02-25 08:48:40
@@ -43,7 +43,6 @@
 import javafx.scene.input.KeyEvent;
 import javafx.scene.input.MouseEvent;
 import javafx.util.Callback;
-import com.sun.javafx.PlatformUtil;
 import com.sun.javafx.scene.control.SizeLimitedList;
 import com.sun.javafx.scene.control.inputmap.InputMap;
 import com.sun.javafx.scene.control.inputmap.InputMap.KeyMapping;
@@ -234,13 +233,13 @@
         // create OS-specific child mappings
         // --- mac OS
         InputMap<C> macInputMap = new InputMap<>(control);
-        macInputMap.setInterceptor(event -> !PlatformUtil.isMac());
+        // TODO: macInputMap.setInterceptor(event -> !PlatformUtil.isMac());
         addDefaultMapping(macInputMap, new KeyMapping(new KeyBinding(SPACE).shortcut().ctrl(), e -> toggleFocusOwnerSelection()));
         addDefaultChildMap(tableViewInputMap, macInputMap);
 
         // --- all other platforms
         InputMap<C> otherOsInputMap = new InputMap<>(control);
-        otherOsInputMap.setInterceptor(event -> PlatformUtil.isMac());
+        // TODO: otherOsInputMap.setInterceptor(event -> PlatformUtil.isMac());
         addDefaultMapping(otherOsInputMap, new KeyMapping(new KeyBinding(SPACE).ctrl(), e -> toggleFocusOwnerSelection()));
         addDefaultChildMap(tableViewInputMap, otherOsInputMap);
 
