--- src/javafx.controls/com/sun/javafx/scene/control/behavior/TreeViewBehavior.java	2026-01-27 06:40:46
+++ src/javafx.controls/com/sun/javafx/scene/control/behavior/TreeViewBehavior.java	2026-02-25 09:07:31
@@ -43,7 +43,6 @@
 import javafx.scene.input.KeyEvent;
 import javafx.scene.input.MouseEvent;
 import javafx.util.Callback;
-import com.sun.javafx.PlatformUtil;
 import com.sun.javafx.scene.control.inputmap.InputMap;
 import com.sun.javafx.scene.control.inputmap.InputMap.KeyMapping;
 import com.sun.javafx.scene.control.inputmap.KeyBinding;
@@ -214,7 +213,6 @@
             new KeyMapping(KP_DOWN, e -> selectNextRow()),
 
             new KeyMapping(new KeyBinding(UP).shift(), e -> alsoSelectPreviousRow()),
-            new KeyMapping(new KeyBinding(KP_UP).shift(), e -> alsoSelectPreviousRow()),
             new KeyMapping(new KeyBinding(DOWN).shift(), e -> alsoSelectNextRow()),
             new KeyMapping(new KeyBinding(KP_DOWN).shift(), e -> alsoSelectNextRow()),
 
@@ -228,13 +226,13 @@
         // create OS-specific child mappings
         // --- mac OS
         InputMap<TreeView<T>> macInputMap = new InputMap<>(control);
-        macInputMap.setInterceptor(event -> !PlatformUtil.isMac());
+        // TODO: macInputMap.setInterceptor(event -> !PlatformUtil.isMac());
         addDefaultMapping(macInputMap, new KeyMapping(new KeyBinding(SPACE).shortcut().ctrl(), e -> toggleFocusOwnerSelection()));
         addDefaultChildMap(treeViewInputMap, macInputMap);
 
         // --- all other platforms
         InputMap<TreeView<T>> otherOsInputMap = new InputMap<>(control);
-        otherOsInputMap.setInterceptor(event -> PlatformUtil.isMac());
+        // TODO: otherOsInputMap.setInterceptor(event -> PlatformUtil.isMac());
         addDefaultMapping(otherOsInputMap, new KeyMapping(new KeyBinding(SPACE).ctrl(), e -> toggleFocusOwnerSelection()));
         addDefaultChildMap(treeViewInputMap, otherOsInputMap);
 
